youtube_dl/extractor/peertube.py

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class PeertubeIE(InfoExtractor):
+    _BASE_VIDEO_URL = 'https://peertube.touhoppai.moe/static/webseed/%s-1080.mp4'
+    _BASE_THUMBNAIL_URL = 'https://peertube.touhoppai.moe/static/previews/%s.jpg'
+    IE_DESC = 'Peertube Videos'
+    IE_NAME = 'Peertube'
+    _VALID_URL = r'https?:\/\/peertube\.touhoppai\.moe\/videos\/watch\/(?P<id>[0-9|\-|a-z]+)'
```

Comment:
Invalid.

Reply:
Which line in particular are you referring to as invalid?

Reply:
This exact line.

Reply:
What is exactly wrong with the capture group as it seems correct to me and runs without any errors?